scripts/make_COMPASS_ELM_forcing.py

```diff
@@ -1,247 +1,154 @@
-import xarray
+import xarray as xr
+import pandas as pd
 import numpy as np
-import matplotlib.pyplot as plt
-
-# Set up an ELM domain for seven sites across 2 regions x 3 points (upland, transition, wetland)
-
-landcover_types=[
-    'Upland',
-    'Transition',
-    'Wetland'
-]
-
-regions=[
-    'Lake Erie',
-    'Chesapeake'
-]
-
-sites=[
-    'SERC',
-    'Sweethall',
-    'Goodwin Island',
-    'Moneystump',
-    'Portage River',
-    'Old Woman Creek',
-    'Crane Creek',
-    ]
-
-grid_points=[]
-for site in sites:
-    for point in landcover_types:
-        grid_points.append(site+'_'+point)
-
-# For now we are setting up each land cover type as a separate grid cell. 
-# In the future we could do some of this using topo units within grid cells
+xr.set_options(display_max_rows=10^4)
+pd.set_option('display.max_rows', 50)
+pd.set_option('display.max_columns', 50)
 
+
+#%%--------------------------------------------------------------------------------------
+# SITE COORDS
+
+# Read in site/zone coordinate file
+# synoptic = \
+    (pd.read_csv('../data/raw/transect_coords/compass_synoptic.csv')
+    .assign(grid_points = lambda x: x.site_id + '_' + x.zone_id)
+    .assign(site_cat = 'synoptic')
+    .drop(columns=['distance'])
+     )
+
+# GET DISTANCE FROM NWI PERMANENT WATER BODY
+dist = pd.read_csv('../../data/site_pts/synoptic/synoptic_sites_pts_v2_wdist.csv')[['site_id', 'zone', 'distance']]
```

Comment:
I wasn't able to run/debug the script because I don't have these datasets. Is it possible to add them to the repo?

Reply:
Added data to:  **data/processed/synoptic_site_pts**  in the repo.